src/MUIDataTable.js

```diff
@@ -269,47 +270,62 @@ class MUIDataTable extends React.Component {
   /*
    *  Build the table data used to display to the user (ie: after filter/search applied)
    */
-
-  isRowDisplayed(columns, row, filterList, searchText) {
-    let isFiltered = false,
-      isSearchFound = false;
+  computeDisplayRow(columns, row, rowIndex, filterList, searchText) {
+    let isFiltered = false;
+    let isSearchFound = false;
+    let displayRow = [];
 
     for (let index = 0; index < row.length; index++) {
-      let column = row[index];
+      let columnDisplay = row[index];
+      let columnValue = row[index];
 
       if (columns[index].customRender) {
-        const funcResult = columns[index].customRender(index, column);
-        column =
+        const tableMeta = this.getCustomRenderMeta(rowIndex, index, row, columns[index], this.state.data, {
+          ...this.state,
+          filterList: filterList,
+          searchText: searchText,
+        });
+
+        const funcResult = columns[index].customRender(
+          columnValue,
+          tableMeta,
+          this.updateDataCol.bind(null, rowIndex, index),
+        );
+        columnDisplay = funcResult;
+
+        /* drill down to get the value of a cell */
+        columnValue =
           typeof funcResult === "string"
             ? funcResult
             : funcResult.props && funcResult.props.value
               ? funcResult.props.value
               : column;
```

Comment:
There's an orphaned reference to `column` here. I think that should be `columnValue`?

Maybe add `no-undef` rule to eslint config?

Reply:
Nice catch. I've updated that and also added some additional eslint rules